MarceloQuiroga-EjercicioHerenciaClinica.cpp

```diff
@@ -0,0 +1,324 @@
+// Ejercicio Herencia Clinica.cpp : Este archivo contiene la función "main". La ejecución del programa comienza y termina ahí.
+//
+
+#include <iostream>
+#include <string>
+
+using namespace std;
+
+class Paciente
+{
+protected:
+    string nombre;
+    int edad;
+    float descuento;
+    string tipoPaciente;
+public:
+    Paciente()
+    {
+
+    }
+
+    Paciente(string nombre,int edad,float descuento,string tipoPaciente)
+    {
+        this->nombre = nombre;
+        this->edad = edad;
+        this->descuento = descuento;
+        this->tipoPaciente = tipoPaciente;
+    }
+
+    void MostrarPaciente()
+    {
+        cout << "Paciente: " << nombre << endl;
+        cout << "Edad: " << edad << endl;
+        cout << "Tipo de Paciente: " << tipoPaciente << endl;
+    }
+
+    string getNombre()
+    {
+        return nombre;
+    }
+
+    float CalcularDescuento()
+    {
+        return descuento;
+    }
```

Comment:
Esto es un simple get, a no  ser que realize algun procedimiento se podria utilizar el verbo calcular...  en todo caso, podrias convertirlo en metodo virtual puro si sabes que necesidad un procedimiento que las clases hijas deben implementar
```suggestion
    float GetDescuento()
    {
        return descuento;
    }
```

opcion 2
```suggestion
    virtual float CalcularDescuento() = 0;
```